config/production.france.js

```diff
@@ -5,6 +5,10 @@ const HOST = process.env.HOST || "api.openfisca.fr",
   apiBaseUrl = process.env.API_URL || `https://${HOST}`,
   gitHubProject = "openfisca/openfisca-france",
   gitWebpageUrl = "https://github.com/openfisca/legislation-explorer",
+  piwikConfig = {
+    url: "stats.data.gouv.fr",
+    siteId: 4
```

Comment:
I added it in 592ebb0 for the pages covered by the "react" part of the app.

The jquery/d3 part (/graph page) requires more work and I'm not sure it's worth doing it before we discuss the future of this page.